diff --git a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
index 908626b..026dda8 100644
--- a/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
+++ b/node_modules/react-native-animated-ellipsis/dist/AnimatedEllipsis.js
@@ -47,7 +47,8 @@ export default class AnimatedEllipsis extends Component {
 
     Animated.timing(this._animation_state.dot_opacities[which_dot], {
       toValue: this._animation_state.target_opacity,
-      duration: this.props.animationDelay
+      duration: this.props.animationDelay,
+      useNativeDriver: true,
     }).start(this.animate_dots.bind(this, next_dot));
   }
 
