diff --git a/node_modules/react-native-swipeable/lib/index.js b/node_modules/react-native-swipeable/lib/index.js
index 57996ad..29a945c 100644
--- a/node_modules/react-native-swipeable/lib/index.js
+++ b/node_modules/react-native-swipeable/lib/index.js
@@ -78,7 +78,7 @@ var Swipeable = function (_PureComponent) {
     }, _this._unmounted = false, _this._handlePan = _reactNative.Animated.event([null, {
       dx: _this.state.pan.x,
       dy: _this.state.pan.y
-    }]), _this._handleMoveShouldSetPanResponder = function (event, gestureState) {
+    }], { useNativeDriver: false }), _this._handleMoveShouldSetPanResponder = function (event, gestureState) {
       return Math.abs(gestureState.dx) > _this.props.swipeStartMinDistance;
     }, _this._handlePanResponderStart = function (event, gestureState) {
       var _this$state = _this.state,
@@ -294,8 +294,8 @@ var Swipeable = function (_PureComponent) {
   }
 
   _createClass(Swipeable, [{
-    key: 'componentWillMount',
-    value: function componentWillMount() {
+    key: 'UNSAFE_componentWillMount',
+    value: function UNSAFE_componentWillMount() {
       var _props = this.props,
           onPanAnimatedValueRef = _props.onPanAnimatedValueRef,
           onRef = _props.onRef;
